Resources/Prototypes/Entities/Structures/Machines/Computers/nav_map_blips.yml

```
@@ -0,0 +1,56 @@
+  color: Cyan
+  texturePaths:
+  - "/Textures/Interface/NavMap/beveled_circle.png"
+
+# Atmos monitoring console
+- type: navMapBlip
+  id: GasPipeSensor
+  selectable: true
+  color: "#ffcd00"
+  texturePaths:
+  - "/Textures/Interface/NavMap/beveled_star.png"
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)